docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -2625,3 +2625,189 @@ function sitenow_update_9067() {
+  $table = $field_types['body']['table'];
+  $revision_table = $field_types['body']['revision_table'];
+  if ($database->schema()->tableExists($teaser_table) && $database->schema()->tableExists($table)) {
+    $database->query("
+      UPDATE {$teaser_table}, {$table}
+      SET {$table}.body_summary = {$teaser_table}.field_teaser_value
+      WHERE {$table}.entity_id = {$teaser_table}.entity_id
+      AND {$table}.bundle = {$teaser_table}.bundle
+      AND {$table}.revision_id = {$teaser_table}.revision_id
+      AND {$teaser_table}.field_teaser_value <> ''
+      AND ({$teaser_table}.field_teaser_value <> {$table}.body_summary OR {$table}.body_summary IS NULL)
+    ")
+      ->execute();
+    if ($database->schema()->tableExists($teaser_revision_table) && $database->schema()->tableExists($revision_table)) {
+      $database->query("
+        UPDATE {$teaser_revision_table}, {$revision_table}
+        SET {$revision_table}.body_summary = {$teaser_revision_table}.field_teaser_value
+        WHERE {$revision_table}.revision_id = {$teaser_revision_table}.revision_id
+        AND {$revision_table}.bundle = {$teaser_revision_table}.bundle
+        AND {$revision_table}.entity_id = {$teaser_revision_table}.entity_id
+        AND {$teaser_revision_table}.field_teaser_value <> ''
+        AND ({$teaser_revision_table}.field_teaser_value <> {$revision_table}.body_summary OR {$revision_table}.body_summary IS NULL)
+      ")
+        ->execute();
+    }
+  }
+
+  // Updates body_summary for any nodes not already in tables due to empty body.
+  $database->query("
+    INSERT IGNORE INTO {$table} ({$table}.bundle, {$table}.deleted, {$table}.entity_id, {$table}.revision_id, {$table}.langcode, {$table}.delta, {$table}.body_value, {$table}.body_summary, {$table}.body_format)
+    SELECT {$teaser_table}.bundle, {$teaser_table}.deleted, {$teaser_table}.entity_id, {$teaser_table}.revision_id, {$teaser_table}.langcode, {$teaser_table}.delta, '', {$teaser_table}.field_teaser_value, 'filtered_html'
+    FROM {$teaser_table}
+    WHERE entity_id IN (SELECT nid FROM node WHERE nid NOT IN (SELECT DISTINCT entity_id FROM node__body))
+  ")
+    ->execute();
+  $database->query("
+    INSERT IGNORE INTO {$revision_table} ({$revision_table}.bundle, {$revision_table}.deleted, {$revision_table}.entity_id, {$revision_table}.revision_id, {$revision_table}.langcode, {$revision_table}.delta, {$revision_table}.body_value, {$revision_table}.body_summary, {$revision_table}.body_format)
+    SELECT {$teaser_revision_table}.bundle, {$teaser_revision_table}.deleted, {$teaser_revision_table}.entity_id, {$teaser_revision_table}.revision_id, {$teaser_revision_table}.langcode, {$teaser_revision_table}.delta, '', {$teaser_revision_table}.field_teaser_value, 'filtered_html'
+    FROM $teaser_revision_table
+    WHERE entity_id IN (SELECT nid FROM node WHERE nid NOT IN (SELECT DISTINCT entity_id FROM node_revision__body))
+  ")
+    ->execute();
```

Comment:
I think this was my mistake earlier saying they could go after the big `if` statements, but they should probably go inside it, after their respective counterparts above.

I don't think we should have any sites where either table doesn't exist since they all get installed with default content, but still good to be safe.